### ISSUE TITLE
No declaration file

### ISSUE THREAD

**Issue by abbyblachman** [gilmoreorless/moment-timezone-data-webpack-plugin#37]
I get this error when trying to use the plugin: 

```
Could not find a declaration file for module 'moment-timezone-data-webpack-plugin'. '/Users/abbyblachman/Desktop/the19th-wp-theme/node_modules/moment-timezone-data-webpack-plugin/src/index.js' implicitly has an 'any' type.
  Try `npm install @types/moment-timezone-data-webpack-plugin` if it exists or add a new declaration (.d.ts) file containing `declare module 'moment-timezone-data-webpack-plugin';`ts(7016)
```

**Comment by gilmoreorless** [comment]
Yep, there's definitely no declaration file, entirely due to laziness on my part. The projects I wrote this plugin for didn't use TypeScript for the webpack config, so the lack of declarations was never a problem.

I'll take a look at adding one soon. 😃

**Comment by gilmoreorless** [comment]
I've published version 1.5.0 which contains a `.d.ts` declaration file.

I tested the definitions locally in temporary projects using webpack 4 and 5. But I know TS can be fussy about declarations sometimes, so let me know if you run into any problems.
